backend/template.yaml

```diff
@@ -42,6 +42,22 @@ Resources:
       Policies:
         - DynamoDBWritePolicy:
             TableName: !Ref MockConfig
+  GetMock:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Handler: GetMock.handler
+      Environment:
+        Variables:
+          TableName: !Ref MockConfig
+      Events:
+        MockRoute:
+          Type: Api
+          Properties:
+            Method: get
+            Path: '/{proxy+}'
```

Comment:
Should we prefix this with `/get/`? I'm just comparing to the other endpoints.

Reply:
Done